frontend/public/actions/features.ts

```diff
@@ -261,5 +263,6 @@ export const detectFeatures = () => (dispatch: Dispatch) =>
     detectLoggingURL,
     detectConsoleLinks,
     ...ssarCheckActions,
-    ...plugins.registry.getCustomFeatureFlags().map((ff) => ff.properties.detect),
+    ...extensionFeatureFlags.map((ff) => ff.properties.detect),
```

Comment:
In this case, I'd simply inline the whole expression (since it's evaluated just once):

```ts
...pluginStore
  .getAllExtensions()
  .filter(isCustomFeatureFlag)
  .map((ff) => ff.properties.detect),
```